docs/payment/issue-payment-fsp-portal.md

```diff
@@ -4,41 +4,94 @@ hide:
   - toc
 ---
 
-!!! info "This section explains"
-    How to process a manual payment via a FSP portal, export payment instructions and reconcile payments.
+In the 121 Platform, payments can be processed in two different methods: with a manual payment via a FSP portal or directly with a FSP integration. Below, you will find the step-by-step instructions to execute the manual payment via a FSP portal, to export the payment instructions and reconcile payments into your program.
 
+*For payments instructions with an integrated Financial Service Provider, please refer to the instructions on the [Payment with an integrated FSP page](./issue-payment-integrated-fsp.md)*
 
-### Overview of pending payment
+!!! Important "Who can perform actions in this page?"
+    Finance Manager and Finance Officer roles can perform actions in the Payment page.  
+    Additional users may be granted only **View** permission on this page. [Learn more about Users & Roles](../users/users-roles-page.md)
+
+
+### Include People Affected in the payment round
+
+The Payment table shows all of the included households in your program ready to receive payment with the status **Included**.
+To initiate a payment:
+
+- On the top left of the table, Open the drop down to **Choose action.**
+- Select  **Do Payment#**
 
-Explain the payment rounds.
+IMG
 
-### Issue and Review payments to beneficiaries
+- Include the People Affected in the payment round as follows.
+- After selecting **payment#**, a tick box :material-checkbox-blank-outline: appears on the left of the PA name column.
+  - Make an **individual selection** by clicking on the :material-checkbox-outline: on the PA line;
+  - Or a **bulk selection** by clicking on the :material-checkbox-multiple-marked-outline: on the upper line **Select**
+- **Apply Action** button becomes blue and clickable. Click the button to start paying out to the selected PA.
+- A pop-up appears to confirm the sum to be transferred value to each household and the **Maximum amount that will be spend this payment: SUM.**, matching the total sum of the transfers.
+- Click on **Start paying now**.
+- A message indicates the completion of the action and **next steps to download the payment instructions**.
 
-Select beneficiaries. Bulk or individual.
+Only PAs who have not received the **payment#** round are available for selection. PA’s that have received the max payments will not be applicable (max payment number per household might change as the program continues).
 
-### Export payment instructions
+### Export the payment instructions
```

Comment:
I would actually remove 'the'

Reply:
@cassandradamico why did you resolve this? I don't see it updated in the branch?

Reply:
@tijsziere it is corrected on my side, weird

Reply:
@cassandradamico maybe commit your latest changes? Have you saved all your files?